tests/integration/test_keeper_reconfig_add/configs/keeper2.xml

```diff
@@ -0,0 +1,21 @@
+<clickhouse>
+    <keeper_server>
+        <enable_reconfiguration>true</enable_reconfiguration>
+
+        <tcp_port>9181</tcp_port>
+        <server_id>2</server_id>
+        <log_storage_path>/var/lib/clickhouse/coordination/log</log_storage_path>
+        <snapshot_storage_path>/var/lib/clickhouse/coordination/snapshots</snapshot_storage_path>
+
+        <coordination_settings>
+            <operation_timeout_ms>5000</operation_timeout_ms>
+            <session_timeout_ms>10000</session_timeout_ms>
+            <raft_logs_level>trace</raft_logs_level>
+        </coordination_settings>
+
+        <raft_configuration>
+            <server> <id>1</id> <hostname>node1</hostname> <port>9234</port> </server>
+            <server> <id>2</id> <hostname>node2</hostname> <port>9234</port> </server>
```

Comment:
all configs can start with only that node defined
e.g. for node 2
```xml
   <raft_configuration>
            <server> <id>2</id> <hostname>node2</hostname> <port>9234</port> </server>
    </raft_configuration
```

for node 3
```xml
   <raft_configuration>
            <server> <id>3</id> <hostname>node3</hostname> <port>9234</port> </server>
    </raft_configuration
```

Reply:
No, they can't, as they form a single node cluster and everything messes up. As is Zookeeper, let's assume we start new nodes with new cluster config, not in single mode